jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/internal/FrameFlusher.java

```diff
@@ -353,7 +307,7 @@ private void release()
         @Override
         public String toString()
         {
-            return String.format("%s[%s,%s,%b,%s]", getClass().getSimpleName(), frame, callback, batch, terminated);
+            return String.format("%s{%s,%s,%b}", getClass().getSimpleName(), frame, callback, batch);
         }
     }
 }
```

Comment:
This has been improved, but I think there are still some concerns.
@sbordet @lachlan-roberts 
The previous `processError` has now been split into `processHandlerError` and `processConnectionError` in an attempt to distinguish errors that come from the application(aka Handler) vs those from the network.

Handler errors are just converted to a close status and a close frame is sent.  The only difference to normal handling is that a close response is not waited for before `onClosed` is called on the handler.   I think that either we should wait (ie send a close and wait for a response) OR if we are not waiting then we need a forced close that always puts us into `CLOSED` state (rather than `OSHUT`) and closes the connection as soon as the frame is sent.

Connections errors for the most part do not try to send a close frame (with the exception of protocol errors that we try to signal to the other end).  The application is informed with an `onClosed` call and then the connection is closed. Again perhaps a forced change to `CLOSED` state shouild be done?